src/main/java/mekanism/common/block/BlockBasic.java

```diff
@@ -270,9 +270,29 @@ public float getExplosionResistance(World world, BlockPos pos, Entity exploder,
         IBlockState state = world.getBlockState(pos);
         BasicBlockType type = BasicBlockType.get(getBasicBlock(), state.getBlock().getMetaFromState(state));
         if (type == BasicBlockType.REFINED_OBSIDIAN) {
-            return 4000F;
+            return 2400F;
         }
-        return blockResistance;
+        else if (type == BasicBlockType.OSMIUM_BLOCK) {
```

Comment:
This would probably look cleaner as a `switch`

Reply:
Yeah it would, just didn't made one as i first only added a second if.
I will change that